openbb_terminal/cryptocurrency/onchain/onchain_controller.py

```diff
@@ -427,6 +430,50 @@ def call_whales(self, other_args: List[str]):
                 else None,
             )
 
+    @log_start_end(log=logger)
+    def call_topledger(self, other_args: List[str]):
+        """Process topledger command"""
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="topledger",
+            description="""
+                    Display on-chain data from topledger.
+                    [Source: https://docs.topledger.xyz/]
+                """,
+        )
+
+        parser.add_argument(
+            "-o",
+            "--org",
+            dest="org_slug",
+            type=str,
+            help="Organization Slug [tensor]",
+            default=None,
+        )
+        parser.add_argument(
+            "-q",
+            "--query",
+            dest="query_slug",
+            type=str,
+            help="Query Slug [daily-transactions/weekly-transactions/monthly-transactions"
+            "/daily-users/monthly-users/weekly-users/daily-gmv-sol/weekly-gmv/"
+            "monthly-gmv/daily-tvl/weekly-tvl/monthly-tvl/top-traders-by-gmv]",
+            default=None,
```

Comment:
we can add the choices as an array to get some sweet autocompletions

like 
```
choices=["daily-transactions", "...."]
```

Reply:
have updated help text and options under choices.